src/test/features/response/routes/how-much-paid-claimant.ts

```diff
@@ -0,0 +1,153 @@
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.successful.withText('How much have you paid the claimant?'))
+        })
+      })
+    })
+  })
+
+  describe('on POST', () => {
+    const method = 'post'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'citizen')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+
+      context('when response not submitted', () => {
+        it('should redirect to response type page when response type is not full admission', async () => {
+          draftStoreServiceMock.resolveFind('response', { response: { type: ResponseType.PART_ADMISSION } })
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+
+          await request(app)
+            .post(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.redirect.toLocation(ResponsePaths.responseTypePage
+              .evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })))
+        })
+
+        context('when form is invalid', () => {
+          it('should return 500 and render error page when cannot retrieve claim', async () => {
+            claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+            await request(app)
+              .post(pagePath)
+              .set('Cookie', `${cookieName}=ABC`)
+              .expect(res => expect(res).to.be.serverError.withText('Error'))
+          })
+
+          it('should render page when everything is fine', async () => {
+            draftStoreServiceMock.resolveFind('response', draftOverride)
+            claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+
+            await request(app)
+              .post(pagePath)
+              .set('Cookie', `${cookieName}=ABC`)
+              .expect(res => expect(res).to.be.successful.withText('How much have you paid the claimant?', 'div class="error-summary"'))
+          })
+        })
+
+        context('when form is valid', () => {
+          it('should return 500 and render error page when cannot save draft', async () => {
+            claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+            draftStoreServiceMock.resolveFind('response', draftOverride)
+            draftStoreServiceMock.rejectSave()
+
+            await request(app)
+              .post(pagePath)
+              .set('Cookie', `${cookieName}=ABC`)
+              .send({ option: 'amountClaimed' })
+              .expect(res => expect(res).to.be.serverError.withText('Error'))
+          })
+
+          it('should redirect to how much paid task list page when everything is fine', async () => {
```

Comment:
Please add missing test case for a redirect to 'send response by email' when defendant claims that less then claim value was paid. See:

```
if (draft.document.howMuchPaidClaimant.option === HowMuchPaidClaimantOption.LESS_THAN_AMOUNT_CLAIMED) {
  res.redirect(Paths.sendYourResponseByEmail.evaluateUri({ externalId: externalId })) // TEST THAT!!!
} else {
  res.redirect(Paths.taskListPage.evaluateUri({ externalId: externalId }))
}
```